todo_app/templates/index.html

```diff
@@ -11,8 +11,31 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+        {% for item in getlist%}
+		<li> {{item.id}} - {{item.title}} - {{item.status}} 
+      {% if item.status == "Not Started"%}
+      <form method="POST" action = "/complete">
+        <input type="hidden" name="itemcomplete" value = "{{item.id}}"> 
+        <input type="submit" value="Complete" class="btn btn-success">         
+      </form>
+      {%endif%}
+      <form method="POST" action = "/delete">
+        <input type="hidden" name="itemdelete" value = "{{item.id}}"> 
+        <input type="submit" value="Delete" class = "btn btn-danger"> 
+      </form>
+    </li>
+		{% endfor%}
```

Comment:
Even though HTML doesn't care about whitespace, it still makes it easier to read if your indentation matches the structure.